app/containers/PriceDisplay/PriceDisplay.jsx

```diff
@@ -0,0 +1,40 @@
+// @flow
+import React, { Component } from 'react'
+import { formatFiat } from '../../core/formatters'
+import styles from './PriceDisplay.scss'
+import classNames from 'classnames'
+
+type Props = {
+  neoPrice: number,
+  gasPrice: number
+}
+
+export default class PriceDisplay extends Component<Props> {
+  render () {
```

Comment:
I think this could be better written as a functional component - 
`const PriceDisplay = ({ ... }: Props) => { ... return (...) }`

Reply:
Also, could you move the PriceDisplay to the components section? Then from the dashboard container pass the props it needs.